SPECS/kernel/config_aarch64

```diff
@@ -8777,10 +8776,8 @@ CONFIG_COMMON_CLK_MT8192=y
 # CONFIG_COMMON_CLK_MT8192_VENCSYS is not set
 CONFIG_COMMON_CLK_MT8195=y
 CONFIG_COMMON_CLK_MT8195_APUSYS=y
-CONFIG_COMMON_CLK_MT8195_AUDSYS=y
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/3d02397ca30b25b778f3a7cb823f25b854a84380

This config does nothing in the source